graylog2-server/src/main/java/org/graylog2/shared/rest/documentation/generator/Generator.java

```diff
@@ -81,18 +85,37 @@
     private static Map<String, Object> overviewResult = Maps.newHashMap();
     private static Reflections reflections;
 
+    private final Map<String, String> packageNames;
     private final ObjectMapper mapper;
 
-    public Generator(String[] packageNames, ObjectMapper mapper) {
+    public Generator(Map<String, String> packageNames, ObjectMapper mapper) {
+        this.packageNames = packageNames;
         this.mapper = mapper;
 
         if (reflections == null) {
-            reflections = new Reflections((Object[]) packageNames);
+            reflections = new Reflections((Object[]) packageNames.keySet().toArray());
         }
     }
 
     public Generator(String packageName, ObjectMapper mapper) {
-        this(new String[]{packageName}, mapper);
+        this(ImmutableMap.of(packageName, ""), mapper);
+    }
+
+    public String prefixedPath(Class<?> resourceClass, @Nullable String annotationPath) {
```

Comment:
This method should be `private` or at least `protected` (if we had a test for it).